### PR TITLE
Added initial keldata map and tiled editor support

### PR DESCRIPTION
**Pull request by SgtFlame**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,20 +15,24 @@
   "author": "Excalibur.js Team",
   "license": "BSD-2-Clause",
   "devDependencies": {
+    "@excaliburjs/testing": "0.25.1",
     "clean-webpack-plugin": "3.0.0",
     "compression-webpack-plugin": "7.1.2",
+    "file-loader": "^6.2.0",
     "html-webpack-plugin": "5.5.0",
-    "@excaliburjs/testing": "0.25.1",
+    "raw-loader": "^4.0.2",
```
Comment (SgtFlame) [inline package.json:23]: I'm not sure if this is needed.  See if it can be removed.

File: package.json [hunk 2]
```diff
@@ -15,20 +15,24 @@
   "author": "Excalibur.js Team",
   "license": "BSD-2-Clause",
   "devDependencies": {
+    "@excaliburjs/testing": "0.25.1",
     "clean-webpack-plugin": "3.0.0",
     "compression-webpack-plugin": "7.1.2",
+    "file-loader": "^6.2.0",
```
Comment (SgtFlame) [inline package.json:21]: I'm not sure if this is needed.  See if it can be removed.

File: src/index.ts [hunk 3]
```diff
@@ -1,36 +1,75 @@
...
+  Loader,
+  DisplayMode,
+  IsometricMap,
+  Vector,
+  vec,
+  ImageSource,
+  Resource,
+  Logger,
+} from 'excalibur';
+// import { LevelOne } from './scenes/level-one/level-one';
```
Comment (SgtFlame) [inline src/index.ts:13]: Remove commented code.

File: webpack.common.js [hunk 4]
```diff
@@ -13,6 +13,36 @@ module.exports = {
...
+      // {
+      //   test: /\.tsj$/,
+      //   type: "asset/resource",
+      //   include: [
+      //     path.resolve(__dirname, 'tilesets'),
+      //   ],
+      //   generator: {
+      //     filename: '[name][ext]'
+      //   }
+      // },
```
Comment (SgtFlame) [inline webpack.common.js:35]: Remove this section

File: tiled/images/Floor_Metal_01-128x64.png
Comment (goznorlives) [inline tiled/images/Floor_Metal_01-128x64.png:1]: Is there something up with this file? It's saying "Unable to render rich display"....

Comment (SgtFlame) [reply tiled/images/Floor_Metal_01-128x64.png:1]: Where does it say this?

**Pull request merged**